packages/mobile/src/screens/profile-screen/ProfileHeader/ProfileSocials.tsx

```diff
@@ -64,10 +72,11 @@ export const ProfileSocials = () => {
         handle: instagram_handle,
         SocialLink: InstagramSocialLink
       },
-      { type: 'tiktok', handle: tiktok_handle, SocialLink: TikTokSocialLink }
+      { type: 'tiktok', handle: tiktok_handle, SocialLink: TikTokSocialLink },
+      { type: 'website', handle: website, SocialLink: WebsiteSocialLink }
```

Comment:
whoa really haha. How long has this been missing?

Reply:
i think they updated the designs somewhat recently, so went ahead and added this since we are here